cloudtests.py

```diff
@@ -60,18 +60,44 @@ def test_net_name(self):
 
 
 class TestJournalWritten(unittest.TestCase):
+    """
+    Test to check that journal logs get written to disk(/var)
+    and make sure that user-uid.journal is not lost on first boot
+    the test should run on First Boot
+    https://github.com/kushaldas/tunirtests/issues/28
+    https://github.com/kushaldas/tunirtests/issues/48
```

Comment:
i was referring more to linking to the BZ's but linking to the issues that link to the BZs can be ok too